### ISSUE TITLE
fixed header hides the right side elements

### ISSUE THREAD

**Issue by boussadjra** [azouaoui-med/css-pro-layout#3]
When I fix the header using `fixed-header` class the header elements right aligned are hidden, as workaround I did :

```
.layout.fixed-header .header {
    width: calc(100% - 280px);
}
```

**Comment by azouaoui-med** [comment]
Can you show the layout type you used ?

**Comment by boussadjra** [comment]
it's happens when I add `fixed-header` and `fixed-sidebar`  classes to the layout in this [example](https://codesandbox.io/s/css-pro-layout-vue-forked-b10mm?file=/src/App.vue) the header takes the whole width, but in my other example when I collapse the sidebar the header looses 280px in the right side and shift to the left.

**Comment by azouaoui-med** [comment]
fixed in [v1.1.0](https://github.com/azouaoui-med/css-pro-layout/releases/tag/v1.1.0)
